sim/make/scala.mk

```diff
@@ -28,3 +28,22 @@ scalafmtAll:
 		firesimLib / scalafmtAll; \
 		midas / scalafmtAll ; \
 		targetutils / scalafmtAll ;"
+
```

Comment:
Maybe it's wise to add a phony target that runs both scalaFix and scalaFmt? 

Do you think it would be sensible to just lump Fix + Fmt together in a single target? Then you could piggyback on the existing CI

Reply:
Forgive my ignorance but I though the existing formatting, for c for example, was a fix and check target where ci only runs check

Reply:
Alright, after re-reading what you wrote, I understand your intentions. I've grouped made two phony targets, `scalaFixFmt` and `scalaFixFmtCheck`. The CI runs `scalaFixFmtCheck`